routemaster/services/logger.rb

```diff
@@ -0,0 +1,74 @@
+require 'logger'
```

Comment:
if you're requiring `logger` then, to be consistent, you should also require 'delegate'